workflow/labels.md

```diff
@@ -37,7 +37,7 @@ Each review label has 3 steps:
 
 | Label   | Description                                                                            |
 | ----    | ---                                                                                    |
-| Blocked | The issue is block by another issue (put it in the description)                        |
+| Blocked | Alerts managers that unblocking it requires their help. Any blocked issue should have a comment explaining why it’s blocked. Don’t mark it as blocked if, for example, it just needs a meeting to discuss it; that’s something you can arrange yourself. An issue may be blocked in any unclosed Pipeline. |
```

Comment:
This is quite long.

I don't think blocked is always related to managers. Often someone might be blocked waiting for external people without it necessarily needing managers help. For example, we often have things blocked waiting for an RT from IS. This is often not something managers can or should help with.

I also don't think we need "an issue may be blocked in any unclosed Pipeline" - although it's interesting right now as it's different from our current process, it is just the normal way that labels work, so I don't think it needs specifying.

I think the point about arranging your own meetings is just one example of how blockers could be resolved. Maybe we need another section somewhere in our guidelines about ways to resolve common blockers. I'm not sure this table is the place for it.

How about simply:

> Work cannot continue until something is resolved that's outside the assignee's control. The issue should have a comment explaining the blocker.

Reply:
> For example, we often have things blocked waiting for an RT from IS. This is often not something managers can or should help with.

If you’re confident IS will do the work in time, it isn’t blocked in any sense that others need to know about (though you might mention it at pipeline-based standups). If you aren’t confident, either it should be rescheduled, which implies management approval; or it should be escalated, which also involves management (“[please ask your manager to increase the priority of the request within the RT system](https://wiki.canonical.com/InformationInfrastructure/IS#Escalating_Requests)”).

> I also don't think we need "an issue may be blocked in any unclosed Pipeline"

Fair enough, removed.

> I think the point about arranging your own meetings is just one example of how blockers could be resolved.

I included that only as a common example of something that is not actually blocked. Happy to remove it though.